lerna: add page (#7742)
tldr-pages/tldr: pages/common/lerna.md

# lerna

> A tool for managing JavaScript projects with multiple packages.

> > More information: <https://lerna.js.org/>.

Reviewer: ```suggestion
> More information: <https://lerna.js.org>.
```

> - Initialize project files (`lerna.json`, `package.json`, `.git`, …):

Reviewer: Don't use special chars like `…` could be not compatible with the user's terminal.
```suggestion
- Initialize project files (`lerna.json`, `package.json`, `.git`, .etc):
```

Reviewer 2: @navarroaxel right, but it should be `etc.` not `.etc`:
```suggestion
- Initialize project files (`lerna.json`, `package.json`, `.git`, etc.):
```

Author: Why not `...`?

Reviewer: because `...` is for a pattern, like the sequence `1, 2, ...`

`lerna init`

- Install all external dependencies of each package and symlink together local dependencies:

`lerna bootstrap`

> - Run `npm run {{script}}` for every package that contains that `{{script}}`:

Reviewer: Perhaps this better
```suggestion
- Run a specific script for every package that contains that script in the `package.json`:
```
